packages/website/src/pages/navigation/Tabs.tsx

```diff
@@ -95,7 +39,60 @@ const WithIconsExmaple: React.FunctionComponent = () => (
             </TabPaneConnected>
         </TabContent>
     </>
+    );
+`;
+
+const code = `

```

Comment:
This example looks pretty bad on my mac screen, maybe we should remove a few tabs?

<img width="685" alt="image" src="https://user-images.githubusercontent.com/260007/157950942-a237970f-81ee-4993-af74-cc8108bd6bf0.png">